index.html

```diff
@@ -1,44 +1,44 @@
-        rel="stylesheet">
-        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"
-        integrity="sha384-oS3vJWv+0UjzBfQzYUhtDYW+Pj2yciDJxpsK1OYPAYjqT085Qq/1cq5FLXAZQ7Ay" crossorigin="anonymous" />
-    </head>
-
-<body>
-    <header class="nav-bar">
-        <a id="logo" href="#" class="logoname">Ritika Rawat</a>
-        <button id="menu" type="button"></button>
-    </header>
-    <section class="first-sec">
-        <h1>Hey There.<br>I'am Ritika</h1>
-        <h2>I'am Software Developer</h2>
-        <p>I can help you build a product, feature or website. Look through some of my work and experience! If you like
-            what
-            you see and have a project you need coded, don't hesitate to contact me.</p>
-        <ul class="social-media-icons">
-            <li>
-                <a href="https://github.com/ritikarawat220" target="_blank" rel="noopener noreferrer"><img
-                        src="./images/github.svg" alt="GitHub"></a>
-            </li>
-            <li><a href="www.linkedin.com/in/rawatritika" target="_blank" rel="noopener noreferrer"><img src="./images/linked-in.svg"
-                        alt="LinkedIn"></a>
-            </li>
-            <li><a href="#" target="_blank" rel="noopener noreferrer"><img src="./images/p.svg" alt="AngelList"></a>
-            </li>
-            <li><a href="https://twitter.com/Ritikarawat22" target="_blank" rel="noopener noreferrer"><img
-                        src="./images/twitter.svg" alt="Twitter"></a></li>
-            <li><a href="#" target="_blank" rel="noopener noreferrer"><img src="./images/m.svg" alt="Medium"></a></li>
-        </ul>
-    </section>
-</body>
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <link rel="stylesheet" href="style.css">
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        
+        <title>Myportfolio</title>
+        <link
+        href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Inter:wght@100;200;300;400;500;600;700;800;900&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+        rel="stylesheet">
+        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"
+        integrity="sha384-oS3vJWv+0UjzBfQzYUhtDYW+Pj2yciDJxpsK1OYPAYjqT085Qq/1cq5FLXAZQ7Ay" crossorigin="anonymous" />
+    </head>
+
+<body>
+    <header class="nav-bar">
+        <a id="logo" href="#" class="logoname">Ritika Rawat</a>
+        <button id="menu" type="button"></button>
```

Comment:
- [ ] Kindly wrap this button in a `nav` tag for better semantics 👍 
- [ ] Kindly add the mobile menu icon to this button using icons like font awesome or the image from the template 👍 
- [ ] Kindly reduce the `height` of the navbar to match the one in Figma 💯 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/211841348-638b6266-abd8-4f0a-9e9c-a96e674990b1.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/211841400-aa95303e-8658-4d73-858f-8b58e4203b04.png)